owncloudApp/src/main/java/com/owncloud/android/ui/fragment/PublicShareDialogFragment.java

```diff
@@ -745,11 +746,51 @@ private void updateInputFormAccordingToServerCapabilities() {
                             mCapabilities.getFilesSharingPublicExpireDateDays()));
         }
 
+        // Set password label when opening the dialog
+        if (mReadOnlyButton.isChecked() &&
+                mCapabilities.getFilesSharingPublicPasswordEnforcedReadOnly().isTrue() ||
```

Comment:
What is this ? `.isTrue()` Hard stuff !

Reply:
isTrue() checks if the capability value is TRUE(1)